js/components/modalOverlay.js

```diff
@@ -18,10 +18,19 @@ class ModalOverlay extends ImmutableComponent {
       close = <button type='button' className='close pull-right' onClick={this.props.onHide}><span>&times;</span></button>
       title = <div className='sectionTitle' data-l10n-id={this.props.title} />
     }
-    return <div className='dialog'>
-      {close}
-      {title}
-      {this.props.content}
+    let customTitleClassesStr = (this.props.customTitleClasses ? this.props.customTitleClasses : '')
+
+    return <div className={'dialog ' + customTitleClassesStr}>
```

Comment:
Wow, glad you caught this :-).
Fixed by https://github.com/brave/browser-laptop/pull/3391/commits/2a729d9aacc94127d7bddb5338449cd915235f8c#diff-4b50ff14dadc9756719a577a72f520d6R513